src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -127,7 +127,7 @@ public void ValidPathWithTrailingSlash()
         }
 
         [ConditionalFact(nameof(UsingNewNormalization))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```

Comment:
Could you please open the issue and point me to it so that I can instead of using SkipOnTargetFramework use `[ActiveIssue(issue#, TargetFrameworkMonikers.Uap)]` ? Does this apply to all the tests under System.IO.FileSystem that are being touched?